exporter/cortexexporter/DESIGN.md

```diff
@@ -0,0 +1,287 @@
+
+# **OpenTelemetry Collector Cortex Exporter Design**
+
+Authors: @huyan0, @danielbang907
+
+Date: July 30, 2020
+
+## **1. Introduction**
+
+Cortex is an open source, horizontally scalable, highly available, multi-tenant, long term storage for Prometheus. Cortex accepts data defined by the Prometheus Remote Write API. Our project is focused on developing an exporter for the OpenTelemetry Collector to a Cortex instance. 
+
+The following diagram is the architecture of Cortex. 
+
+![Cortex Archietecture](https://raw.githubusercontent.com/open-o11y/opentelemetry-collector/design-doc/exporter/cortexexporter/cortex.png)
+
+### **1.1 Remote Write API**
+
+Cortex has a Remote Write API that accepts incoming metrics. This exporter should write metrics to a remote URL in a snappy-compressed, protocol buffer encoded HTTP request defined by the Remote Write API. Each request encodes multiple Cortex TimeSeries, which are composed of a set of labels and a collection of samples. Each label contains a name-value pair of strings, and each sample contains a timestamp-value number pair.

```

Comment:
"snappy-compressed" should be a configuration correct?

Reply:
It would make sense but I think the Prometheus server now only supports snappy, thus restricting the choice of the Remote Write API